Yoast/javascript
# Add czech full text tests papers

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds Czech full text tests papers.

## Relevant technical choices:

* The accompanied premium-configuration with same name should be deleted when this PR is merged.

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test` and make sure that all Czech papers pass.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-626
